pkg/image/prune/imagepruner.go

```diff
@@ -190,6 +190,7 @@ func addImageStreamsToGraph(g graph.Graph, streams *imageapi.ImageStreamList, al
 
 		glog.V(4).Infof("Adding ImageStream %s/%s to graph", stream.Namespace, stream.Name)
 		isNode := imagegraph.EnsureImageStreamNode(g, stream)
+		imagegraph.EnsureAllImageStreamTagsNode(g, stream)
```

Comment:
> Is that data used anywhere?

Not yet, but maybe someday.  Hence trying to avoid hamstringing future me during code reviews by having to ensure that certain nodes are added before others.  Doing it this way makes it very easy to allow parallel node creation, which is very likely to come in for `oc status` in the near future to reduce latency.